diff --git a/build-worker.ts b/build-worker.ts
index 5e2a6d5a..ee66871c 100644
--- a/build-worker.ts
+++ b/build-worker.ts
@@ -1,8 +1,8 @@
 import { build } from "bun";
 import { $ } from "bun";
-import ora from "ora";
+// import ora from "ora";

-const buildSpinner = ora("Building").start();
+// const buildSpinner = ora("Building").start();

 await $`rm -rf dist && mkdir dist`;

@@ -18,7 +18,7 @@ await build({
     minify: false,
 });

-buildSpinner.text = "Transforming";
+// buildSpinner.text = "Transforming";

 // Copy Drizzle migrations to dist
 await $`cp -r drizzle dist/drizzle`;
@@ -30,4 +30,4 @@ await $`cp -r node_modules/@img/sharp-linuxmusl-* dist/node_modules/@img`;

 await $`cp -r node_modules/detect-libc dist/node_modules/`;

-buildSpinner.stop();
+// buildSpinner.stop();
